styles/components/_listingCard.scss

```diff
@@ -3,6 +3,9 @@
   height: 283px;
   font-size: $tx5b;
   cursor: pointer;
+  position: relative;
+  bottom: 0;
+  transition: all .2s ease;
```

Comment:
Yes, that should be possible.

On general comment, don't put colors on anything in any file except theme. If you do (like the border color on the listing card), that color cannot be changed by the user's custom theme color.

We'll have to make a more generic "change from x to y border color on hover" class in theme.scss for the listing card. I'll do that in my patch.